config/prod.env.js

```diff
@@ -23,5 +24,6 @@ module.exports = {
   STELLAR_ENV:      JSON.stringify(envVars.STELLAR_ENV),
   IDM_PLUGIN_JS:    JSON.stringify(envVars.IDM_PLUGIN_JS),
   IDM_PLUGIN_TOKEN: JSON.stringify(envVars.IDM_PLUGIN_TOKEN),
-  MAILGUN_SENDER_FROM: JSON.stringify(envVars.MAILGUN_SENDER_FROM)
+  MAILGUN_SENDER_FROM: JSON.stringify(envVars.MAILGUN_SENDER_FROM),
+  STELLAR_TESTNET: JSON.stringify(envVars.STELLAR_TESTNET)
```

Comment:
@michielmulders We have the `STELLAR_ENV` already which signifies the selection of testnet or public. We should use that to avoid problems.